docs/working-with-projects/working-with-images/create-tag-retention-rules.md

```diff
@@ -3,19 +3,19 @@ title: Create Tag Retention Rules
 
 ## How Tag Retention Rules Work
 
 You define tag retention rules on repositories, not on projects. This allows for greater granularity when defining your retention rules. As the name suggests, when you define a retention rule for a repository, you are identifying which tags to retain. You do not define rules to explicitly remove tags. Rather, when you set a rule, any tags in a repository that are not identified as being eligible for retention are discarded. 
 
-A tag retention rule has 3 filters that are applied sequentially, as described in the following table.
+A tag retention rule has three filters that are applied sequentially, as described in the following table.
 
 |Order|Filter|Description|
 |---|---|---|
-|First|Repository or repositories|Identity the tags on which to apply the rule. You can identify repositories that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*repo`, `repo*`, and `**`) are permitted. The repository filter is applied first to mark the repositories to which to apply the retention rule. The identified repositories are earmarked for further matching based on the tag criteria. No action is taken on the nonspecified repositories at this stage.|
+|First|Repository or repositories|Identify the tags on which to apply the rule. You can identify repositories that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards are permitted (for example, `*repo`, `repo*`, and `**`). The repository filter is applied first to mark the repositories to which to apply the retention rule. The identified repositories are marked for further matching based on the tag criteria. No action is taken on the nonspecified repositories at this stage.|
 |Second|Quantity to retain|Set which tags to retain either by specifying a maximum number of tags, or by specifying a maximum period for which to retain tags.|
-|Third|Tags to retain|Identify the tag or tags on which to apply the rule. You can identify tags that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*tag`, `tag*`, and `**`) are permitted. Use the checkbox to select whether untagged artifacts should be captured as part of the set of artifacts eligible for tag retention.|
+|Third|Tags to retain|Identify the tag or tags on which to apply the rule. You can identify tags that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards are permitted (for example, `*tag`, `tag*`, and `**`). Select the checkbox to indicate that untagged artifacts should be captured as part of the set of artifacts eligible for tag retention.|
```

Comment:
@LandoVMW Hi Landon, can you add a note on the 'exclude' modifier? it's causing some confusion, check this ticket  https://github.com/goharbor/harbor/issues/11450. 

also curious what you think about this

Reply:
If it requires an entire side topic to really explain what is happening with a single dropdown selection (and i think it probably does), perhaps we need to fix the interface for how this rule is created. 
I think the biggest issue with this kind of 'rule' is that you don't really "run" a retention rule. That is to say, you are rather defining which tags get kept when the rest of the tags are cleaned up (deleted).
And the second big issue is that if you "exclude" a repository, it gets ignored ... but if you "exclude" a tag, it gets deleted. These are two contradictory uses of "exclude" in the same relatively small modal dialog.

Reply:
Agreed, for the purpose of this PR, please add a single line that 'exclude' tag equates to delete.  Your understanding is correct.

Reply:
Added a clarifying note after the table.